src/main.cpp

```diff
@@ -3004,6 +3004,22 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, CBlockIn
         }
     }
 
+    // Coinbase transaction must include founders' reward if the
+    // block height is below 210000, which is the first halvening.
```

Comment:
Note the constant of 210000 blocks is correct now for the given block target rate.

However, we intend to change the block target time _and_ monetary base reward to be lower, then to reciprocally increase the halving intervals so that the curve looks like Bitcoin's with (roughly) the same time-since-launch scale.